oneanddone/tasks/migrations/0022_make_taskname_unique.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        changed = set()
+        for task in orm.Task.objects.all():
+            value = getattr(task, 'name')
```

Comment:
Just wondering, why did you choose to use `getattr` and `setattr`, rather than just using the built in getter and setter for the object? I.e., this line could just be `value = task.name`, which is a bit more readable and understandable than `value = getattr(task, 'name')`.